pintracker/ptutil/pin.go

```diff
@@ -0,0 +1,28 @@
+package ptutil
+
+import (
+	"github.com/ipfs/ipfs-cluster/api"
+
+	logging "github.com/ipfs/go-log"
+	peer "github.com/libp2p/go-libp2p-peer"
+)
+
+var logger = logging.Logger("pintrackerutil")
```

Comment:
I think go-log wont truncates long name. Might wanna go with `pintrckutil` or something shorter.

Actually, It doesn't make me super happy to have more logging facilities around. I would just not log anything here. Perhaps we can simply not log that message or log it somewhere else. Otherwise, this need to be added to the list of facilities in `logging.go`

Reply:
Or just return `(bool, error)`?

Reply:
scratch that idea...